src/Modules/Settings/Dnn.PersonaBar.Extensions/Services/ExtensionsController.cs

```diff
@@ -982,8 +983,8 @@ public HttpResponseMessage GetModuleFiles(string ownerFolder, string moduleFolde
                         files.AddRange(GetFiles(Globals.HostMapPath + "Templates\\", ".module.template"));
                         break;
                     case FileType.Manifest:
-                        files.AddRange(GetFiles(folder, "*.dnn"));
-                        files.AddRange(GetFiles(folder, "*.dnn5"));
```

Comment:
Also, please move the regex to a static readonly field and avoid single-character variables, even in lambdas.